data/abilities.ts

```diff
@@ -2720,7 +2720,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 	opportunist: {
 		onFoeAfterBoost(boost, target, source, effect) {
 			// Prevent an infinite loop of oppritunist boosts
-			if (effect?.id === 'opportunist') return;
+			if (effect?.id.endsWith('mirrorherb') || effect?.id.endsWith('opportunist')) return;
```

Comment:
why `endsWith`? Is there something that starts with something else followed by that which is relevant?

Reply:
Alternatively you could check `effect?.name === 'Opportunist'` etc.